reference-app-affinidi-vault.sln

```diff
@@ -0,0 +1,42 @@
+﻿
```

Comment:
What is purpose of this file?

Reply:
I didn't see this file in the repo.

Reply:
okay, but why is it added?

Reply:
this is the solution file for dotnet project, not needed but autogenerates if opened in Visual studio or supported editors. 
REMOVED.